src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -318,17 +335,55 @@ export const useSearch = (services: DiscoverViewServices) => {
         data.search.showError(error as Error);
         return;
       }
+      // TODO: Create a unify error response at server side
```

Comment:
i think this will be important for the next phase. it's not very maintainable if frontend needs to understand and track what the possible error schemas are